src/views/Culture.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="container-fluid">
+    <div class="intro">
+      <h1 id="hook">Culture at CtC</h1>
+      <p class = 'hook-sub'>
+        Code the Change is about more than just technical work—we strive continuously to incorporate our core values in our club culture.
+      </p>
+    </div>
+    <div id="white">
+      <div id="content">
+        <!-- Line breaks in the markdown are treated literally -->
+        <VueShowdown markdown="
+# Our Values
+We commit to upholding the letter and spirit of these values both as individuals and as a community. We hold each other and ourselves accountable for ensuring everything we do at Code the Change represents these values.
+
+## Growth
+
+- We learn from each other and our partners, and we enable others’ learning by sharing our skills, knowledge, and perspectives.
+- We support each other's growth by making space for each other, and embracing our own and each other’s mistakes as opportunities for growth.
+- We give constructive feedback kindly and receive it graciously.
+- We take time to reflect and improve.
+
+## Community
+
+- We respect, value, support, and cherish each other.
+- We welcome members of all backgrounds and abilities, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation (The Contributor Covenant). We have no tolerance for discrimination on the basis of these characteristics, and we accept the responsibility to prevent harassment, abuse, and intimidation of any kind within our organization.
+- We cultivate empathy and kindness for each other, our partners, and the broader community.
+- We are honest and forthright with each other, our partners, and all others with whom we interact. We strive to operate transparently wherever possible.
+
+## Humility
+
+- We forge reciprocal relationships with our partners. We help and learn from each other in a bilateral exchange, not a unilateral movement of aid from us to them.
+- We are fallible, and in fact we will surely make mistakes. We create a space with safeguards and a culture that makes those mistakes okay.
+- We recognize that we need each other’s help, so we ask for help and encourage each other to do likewise.
+
+## Positive Impact
+
+- We commit to ensuring our work has a positive impact for people who need it.
+- We are mindful of the impact our work has, and we accept responsibility for that impact, both good and bad.
```

Comment:
Maybe we should also emphasize that unintentional impact is part of this. For example:
```suggestion
- We are mindful of the impact our work has, and we accept responsibility for that impact, both good and bad, intentional and unintentional.
```

Reply:
Yes, agreed. Added it!